features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -2601,8 +2652,121 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                     );
                                 } }
                                 readOnly={ readOnly }
+                                listen={ (values: Map<string, FormValue>) => handleTokenTypeChange(values) }
                                 data-testid={ `${ testId }-access-token-type-radio-group` }
                             />
+                        { selectedTokenType === "JWT" ? (
```

Comment:
Shall we use a defined constant for "JWT". I think there should already be such a constant.

Reply:
fixed with existing value